host-components/external/enarx-keep@.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Enarx Keep-Loader
+Documentation=https://github.com/enarx/
+
+[Service]
+Type=simple
+Environment="SCRIPT_ARGS=%I"
+ExecStart=/home/mike/programming/enarx/host-components/target/debug/keep-loader $SCRIPT_ARGS
```

Comment:
Why the indirection over `Environment` ??

Reply:
Don't think this will get fixed in time for the end of the Sprint, so new issue here for all systemd-related questions: https://github.com/enarx/enarx/issues/849

Reply:
Looked into this, and it seems that this is required for multiple params.  Leaving for now.